waveform.py

```diff
@@ -310,7 +310,7 @@ def getBaseline(self, n_offset = 0):
         return bl
 
     ## Retrieves RMS of waveform with 100 samples starting from sample 0. Can be tuned
-    def getRMS(self, n_offset = 0):
+    def getRMS(self, n_offset = 5):     #because MANGO corrections sometimes have a wrong first point
```

Comment:
Can you explain better the reason of the offset? It is valid only for MANGO I cannot accept it as it will affect LIME as well. Also then change the comment above as well

Reply:
The 5 sample offset intends to remove spurious points that show up way above/below the baseline. Does not affect the actual signals as they have an horizontal offset (where the signal starts) of 200 and 1500 samples, for the fast and slow digitizers, respectively.